pkg/graph/resolvers/model_plan_phase_test.go

```diff
@@ -0,0 +1,179 @@
+package resolvers
+
+import (
+	"time"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/cmsgov/mint-app/pkg/storage"
+	"github.com/cmsgov/mint-app/pkg/userhelpers"
+
+	"github.com/cmsgov/mint-app/pkg/email"
+	"github.com/cmsgov/mint-app/pkg/graph/model"
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/shared/oddmail"
+)
+
+func (suite *ResolverSuite) TestSendEmailForPhaseSuggestionByModelPlanID() {
+	mockController := gomock.NewController(suite.T())
+	mockEmailService := oddmail.NewMockEmailService(mockController)
+	mockEmailTemplateService := email.NewMockTemplateService(mockController)
+
+	planName := "Plan For Milestones"
+	plan := suite.createModelPlan(planName)
+
+	collaboratorInput := &model.PlanCollaboratorCreateInput{
+		ModelPlanID: plan.ID,
+		UserName:    "CLAB",
+		TeamRoles:   []models.TeamRole{models.TeamRoleLeadership},
+	}
+
+	expectedEmail := "CLAB.doe@local.fake" // This comes from the stub fetch user info function
+
+	_, _, err := PlanCollaboratorCreate(
+		suite.testConfigs.Context,
+		suite.testConfigs.Store,
+		suite.testConfigs.Store,
+		suite.testConfigs.Logger,
+		nil,
+		nil,
+		email.AddressBook{},
+		collaboratorInput,
+		suite.testConfigs.Principal,
+		false,
+		userhelpers.GetUserInfoAccountInfoWrapperFunc(suite.stubFetchUserInfo),
+		false,
+	)
+	suite.NoError(err)
+
+	//Asset that making a collaborator also creates an account
+	account, uAccountErr := storage.UserAccountGetByUsername(suite.testConfigs.Store, collaboratorInput.UserName)
```

Comment:
Is this needed to be tested here? I think this might be more appropriate in the plan collaborator resolver test than here.

Reply:
I thought about this for a while. I think I want to leave the assertion here not  because of the flow assertion but because if things break we understand why. You are correct that the core test for this feature set should be in plan collaborator.